src/utils/combo/filter.ts

```diff
@@ -44,6 +57,10 @@ export const defaultComboFilterSettings: ComboFilterSettings = {
   perCharacterMinComboPercent: {
     [Character.JIGGLYPUFF]: 85,
   },
+  includesComboSequence: {
+    sequence: [],
+    mode: ComboSequenceFilterMode.include,
```

Comment:
```suggestion
    mode: "include",
```
I feel like it's simpler to just use a string instead of an enum.